bin/eslint-transforms.js

```diff
@@ -2,15 +2,44 @@
 
 "use strict";
 
-var nodeCLI = require("shelljs-nodecli");
+var execSync = require("child_process").execSync;
+var objectAssign = require("object-assign");
```

Comment:
Yeah, that's fine. I generally don't like adding code that I intend to remove later (that's how tech debt is created), but since it's mostly up to you already, that's fine. :)